mobify/platform-scaffold
# [Integration Manager] Don't parse for the footer anymore

This PR builds in the navigation links for the footer. While doing this I noticed that the newsletter form data in the Redux store was unused so I removed that too. 

 **JIRA**: N/A

## Changes
- Build in the links for the footer
- Remove footer parser and corresponding actions, selectors, and reducer lines.
- No longer disable the newsletter form while waiting for the data.

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- See that the footer is still full of links.
